trezorlib/tx_api.py

```diff
@@ -32,6 +33,23 @@ def is_capricoin(coin):
     return coin["coin_name"].lower().startswith("capricoin")
 
 
+def is_dash(coin):
+    return coin["coin_name"].lower().startswith("dash")
+
+
+def pack_varint(n):
+    if n == 0:
+        return b"\x00"
+    elif n < 253:
+        return struct.pack("<B", n)
+    elif n <= 65535:
+        return struct.pack("<BH", 253, n)
+    elif n <= 4294967295:
```

Comment:
please use hex constants here (`0xFFFF`, `0xFFFFFFFF`). it makes it clearer on first look that the constants are correct